package.json

```diff
@@ -49,21 +49,22 @@
     "compression-webpack-plugin": "^4.0.0",
     "express": "^4.17.1",
     "express-static-gzip": "^2.1.0",
+    "fromentries": "^1.3.2",
     "graphql": "^15.4.0",
     "helmet": "^4.2.0",
     "html-webpack-plugin": "^4.5.0",
     "http-proxy": "^1.18.1",
     "i18next": "^19.8.3",
     "i18next-browser-languagedetector": "^6.0.1",
+    "lodash.keyby": "^4.6.0",
     "lodash.merge": "^4.6.2",
     "mini-css-extract-plugin": "^1.2.1",
     "optimize-css-assets-webpack-plugin": "^5.0.4",
-    "react": "^16.13.1",
     "pino": "^6.7.0",
     "pino-filter": "^1.0.0",
     "pino-http": "^5.3.0",
     "pino-pretty": "^4.3.0",
-    "sass": "^1.29.0",
```

Comment:
Looks like a rebase change - we were at 1.29, but after this would be a 1.26.10

Reply:
this is caused by mistake in the current `package.json` - see https://github.com/strimzi/strimzi-ui/blob/master/package.json#L66-L68 - `sass` is specified twice.

Reply:
I can fix it to use 1.29 though.